com.github.Qv2ray.appdata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop">
+  <id>com.github.Qv2ray</id>
+  <metadata_license>CC0-1.0</metadata_license>

```

Comment:
The appdata lacks a screenshot, launchable tags. It could use a better description, something different than the summary.